ci/terraform/canary-sign-in-with-ipv.tf

```diff
@@ -0,0 +1,44 @@
+module "canary_sign_in_with_ipv" {
+  source               = "./modules/canary"
+  count                = var.environment == "production" ? 0 : 1
+  environment          = var.environment
+  artifact_s3_location = "s3://${aws_s3_bucket.smoketest_artefact_bucket.bucket}"
+  artefact_bucket_arn  = aws_s3_bucket.smoketest_artefact_bucket.arn
+
+  slack_hook_uri      = var.slack_hook_uri
+  sms_bucket_name     = var.use_integration_env_for_sign_in_journey ? "integration-smoke-test-sms-codes" : local.sms_bucket_name
+  sms_bucket_name_arn = var.use_integration_env_for_sign_in_journey ? "arn:aws:s3:::integration-smoke-test-sms-codes" : local.sms_bucket_name_arn
+
+  canary_handler = "canary-sign-in-with-ipv.handler"
+  canary_name    = "smoke-ipv"
+
+  canary_source_bucket     = aws_s3_bucket_object.canary_source.bucket
+  canary_source_key        = aws_s3_bucket_object.canary_source.key
+  canary_source_version_id = aws_s3_bucket_object.canary_source.version_id
+
+  account_management_url = local.account_management_url
+
+  # not used by this canary, but required by module policy, which is shared
+  sns_topic_pagerduty_p1_alerts_arn = aws_sns_topic.pagerduty_p1_alerts.arn
+  sns_topic_pagerduty_p2_alerts_arn = aws_sns_topic.pagerduty_p2_alerts.arn
+  sns_topic_slack_alerts_arn        = data.aws_sns_topic.slack_events.arn
+  test-services-api-key             = var.test-services-api-key
+  test-services-api-hostname        = var.test-services-api-hostname
+  synthetics-user-delete-path       = var.synthetics-user-delete-path
+
+  # this username and password is the same as is used for the 'base' sign in canary; phone number can be any from a test list (Councourse variable)
+  username                   = var.ipv_smoke_test_username
```

Comment:
If this is the same user why do we need to create another user in `user.tf` ?

Reply:
Thanks James. Deleted the comment as we are in fact using a different user for this (to avoid any possible clashing activity around phone OTP).